src/semantic_csv/casters.cljc

```diff
@@ -33,7 +33,10 @@
    (->int {} x))
   ([{:keys [nil-fill]} x]
    (cond
-     (impl/not-blank? x) (-> x string/trim Double/parseDouble int)
+     (impl/not-blank? x) (-> x string/trim
+                             #?(:clj Double/parseDouble
+                                :cljs js/parseInt)
+                             #?(:clj int))
```

Comment:
Good call here taking advantage of `js/parseInt` taking care of decimal modding 👍